packages/components/src/components/hds/form/super-select/single/base.hbs

```diff
@@ -51,6 +51,7 @@
     @search={{@search}}
     @searchEnabled={{@searchEnabled}}
     @searchField={{@searchField}}
+    @searchFieldPosition="before-options"
```

Comment:
This is already the default value for the `Single` component so we could choose to just not expose this argument but I thought we may want to make it more explicit for clarity. (Let me know what you think though.)

Reply:
I don't mind having it explicitly defined. It also makes it more symmetric with the Multiple.